toolkit/scripts/srpm_pack.mk

```diff
@@ -14,12 +14,22 @@
 SRPM_FILE_SIGNATURE_HANDLING ?= enforce
 
 SRPM_BUILD_CHROOT_DIR = $(BUILD_DIR)/SRPM_packaging
+SRPM_BUILD_LOGS_DIR = $(LOGS_DIR)/pkggen/srpms
 
+toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
+srpm_pack_list_file = $(BUILD_SRPMS_DIR)/pack_list.txt
+
+ifeq ($(strip $(SRPM_PACK_LIST)),)
```

Comment:
Shouldn't this be `ifneq`? It looks like we'd hit this branch when `SRPM_PACK_LIST` is empty. Alternatively, we can switch the contents of each branch.

Reply:
Yes, thanks, fixed